### PR TITLE
CON-2911 Markdown Perfect Number Exercise

### PR DESCRIPTION
**Pull request by zanninso**
[CON-2911](https://01talent.atlassian.net/browse/CON-2911)

[CON-2911]: https://01talent.atlassian.net/browse/CON-2911?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: subjects/java/checkpoints/perfect-number/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+### Usage
+
+Here is a possible `ExerciseRunner.java` to test your class:
+
+```java
+public class ExerciseRunner {
+    public static void main(String[] args) {
+        PerfectNumber perfectNumber = new PerfectNumber();
+
+        // Test case 1
```
Comment (MSilva95) [inline subjects/java/checkpoints/perfect-number/README.md:26]: Not a big fan of those comments, but this is just an opinion, we don't see this in any of the other piscines.

File: subjects/java/checkpoints/perfect-number/ExerciseRunner.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+        // Test case 2
+        int number2 = 28;
+        System.out.println("Is " + number2 + " a perfect number? " + perfectNumber.isPerfectNumber(number2));
+
+        // Test case 3
+        int number3 = 12;
+        System.out.println("Is " + number3 + " a perfect number? " + perfectNumber.isPerfectNumber(number3));
+    }
+}
```
Comment (MSilva95) [inline subjects/java/checkpoints/perfect-number/ExerciseRunner.java:17]: Please format the code.

**Review by MSilva95 — CHANGES_REQUESTED** [review]
Run prettier on the code before pushing.

File: subjects/java/checkpoints/perfect-number/ExerciseRunner.java (hunk 2 above)
Comment (zanninso) [reply subjects/java/checkpoints/perfect-number/ExerciseRunner.java:14]: the code is well formatted what should i format exactly ?

File: subjects/java/checkpoints/perfect-number/README.md (hunk 1 above)
Comment (zanninso) [reply subjects/java/checkpoints/perfect-number/README.md]: removed

**Review by MSilva95 — APPROVED** [review]
LGTM!
You can close the conversation if you feel no need to format the runner.java

**Pull request merged**